soc/arc/snps_nsim/CMakeLists.txt

```diff
@@ -2,13 +2,20 @@
 
 if(COMPILER STREQUAL gcc)
   # GNU compiler options
-  zephyr_compile_options(-mcpu=${GCC_M_CPU} -mno-sdata)
-  zephyr_compile_options_ifdef(CONFIG_CPU_ARCEM -mmpy-option=wlh1)
-  zephyr_compile_options_ifdef(CONFIG_CPU_ARCHS -mmpy-option=plus_qmacw)
-  if(CONFIG_CPU_ARCHS)
-    zephyr_compile_options_ifdef(CONFIG_FPU -mfpu=fpud_all)
+  zephyr_compile_options(-mcpu=${GCC_M_CPU})
+  if(CONFIG_64BIT)
+    # ISA_ARCV3 & 64BIT
+    zephyr_compile_options(-ffixed-r13)
```

Comment:
Do we really need that limitation in place?

Reply:
Fixed, thanks!